Scripts/script-CreateArray.yml

```diff
@@ -0,0 +1,37 @@
+commonfields:
+  id: CreateArray
+  version: -1
+name: CreateArray
+script: |-
+  var data = args.arrayData;
+  var seperator = ',';
+  if (args.separator){
+      seperator = args.separator;
+  }
+  var contextKey = 'array';
+  if (args.contextKey){
+      contextKey = args.contextKey;
+  }
+  var array = data.split(seperator);
```

Comment:
not sure but I would trim white spaces as well just for those who insist on writing `x, y, z`